opencontext_py/tests/test_basic.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+import json
+
+from django.db import connection
+from django.test.utils import setup_test_environment
+from django.test.client import Client
+from django.conf import settings
+
+from opencontext_py.apps.ocitems.octypes import models
+from opencontext_py.apps.searcher.solrsearcher.models import SolrSearch
+from opencontext_py.apps.searcher.solrsearcher.makejsonld import MakeJsonLd
+
+settings.ALLOWED_HOSTS += ['testserver']
+
+def test_hello():
+    """
+    a hello world test -- will get rid of once I'm more confident of understanding all these pieces
+    """
+    assert True
+
+def test_main_page():
+    """
+    load up the homepage
+    """
+
+    client = Client()
+
+    response = client.get('/', follow=True)
+    assert response.status_code == 200
+
+def test_ssearch():
+
+    client = Client()
+
+    response = client.get('/subjects-search/.json?response=geo-project', follow=True)
+    assert response.status_code == 200
+
+def test_project_context():
+
+    client = Client()
+    response = client.get('/contexts/projects/3FAAA477-5572-4B05-8DC1-CA264FE1FC10.json', follow=True)
+    assert response.status_code == 200
+
+def test_index_solr_doc():
+
+    from opencontext_py.apps.ocitems.manifest.models import Manifest
+    from opencontext_py.apps.indexer.reindex import SolrReIndex
+    uuids = ['3FAAA477-5572-4B05-8DC1-CA264FE1FC10']
+    sri = SolrReIndex()
+    sri.reindex_uuids(uuids)
+
+
+def test_other_pages():
+    """
+    a basic way to test functionality of the overall system by loading various pages
+    """
+
+    pages_to_test = [
+        '/about',
+        '/projects-search/',
+        '/subjects-search/#1/-10/13/6/any/Google-Satellite',
```

Comment:
I need to respond to https://github.com/ekansa/open-context-py/pull/741#pullrequestreview-255557856 and then close that issue.  That, delete the stuff after "#" in the test.

Reply:
ok